index.bs

```diff
@@ -2517,6 +2528,9 @@ This [=registration extension=] enables the [RP] to determine which extensions t
 : Client processing
 :: None, except default forwarding of client argument to authenticator argument.
 
+: Client data
+:: Returns the list of supported extensions as a JSON array of extension identifier strings
+
```

Comment:
ClientData (which must be hashed and fed into the authenticator for signature) cannot depend on AuthenticatorData (which comes out of the authenticator after the signature has been computed). In this case it is better to include no client data since the authenticator data will reflect the option anyways.